lib/laa_crime_schemas/types/types.rb

```diff
@@ -279,7 +279,7 @@ module Types
                                              other
                                            ])
 
```

Comment:
Isn't there a risk here that older applications with involvement_in_case set to 'none' blow up ?